assets/css/style.css

```diff
@@ -4,6 +4,7 @@
   Version: 1.0
 */

```

Comment:
I'm not sure why there are so many changes to this file.  I rebased from master and there were big changes in master.  I'm not sure why they're showing up as a diff from master to my branch though.  🤔

Reply:
Do you have your tab set to 4 spaces? I have mine set to 2. It looks like that could be what's causing this.

Reply:
Also there's no newline spacing between CSS rules on the master branch and there is on this one. Do you have a formatting plugin or something maybe that's running automatically on the CSS?